components/ModDashboardTopbar.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+    <div class="flex flex-row items-center justify-between">
+        <h1
```

Comment:
Would it be simpler to just swap out the text based on a switch statement? Or do you think the styling might change based on the page? (Because that would make this approach better )

Reply:
I thought of both ways. I have part of the switch statement for swapping out text.  I decided to put the other two in for the reasons you said. But I feel like for now. If there is a reason to change the styling later maybe a switch statement would be better for now.